sonoff/sonoff.py

```diff
@@ -60,7 +60,7 @@ def do_login(self):
             'version'   : '6',
             'ts'        : int(time.time()),
             'nonce'     : gen_nonce(15),
-            'appid'     : 'oeVkj2lYFGnJu5XUtWisfW4utiN4u9Mq',
+            'appid'     : self._appid,
```

Comment:
Seems like a good change, wonder if it would be worth taking the opportunity to move the other magic strings (os, model, romversion etc) into the __init__ as well.

Reply:
I fixed